keyboards/oddball/rules.mk

```diff
@@ -9,16 +9,16 @@ MCU = atmega32u4
 #   QMK DFU      qmk-dfu
 #   ATmega32A    bootloadHID
 #   ATmega328P   USBasp
-BOOTLOADER = halfkay
+BOOTLOADER = qmk-dfu
```

Comment:
Lines 4-12 should be removed, and `BOOTLOADER = qmk-dfu` should be added to `v2/rules.mk`

Reply:
Done.